src/frontend/src/features/ui/components/app-switcher-panel/index.scss

```
@@ -0,0 +1,178 @@
+  gap: 12px;
+  padding: 2px 4px 10px;
+}
+
+.app-switcher-panel__current-text {
+  display: flex;
+  flex-direction: column;
+  gap: 2px;
+}
+
+.app-switcher-panel__you-are-in {
+  font-family: 'Poppins', system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)